### PR TITLE
[Merged by Bors] - chore(RingTheory/Ideal/Cotangent): Restore lemmasOnly config on `Ideal.toCotangent`.

### PR DESCRIPTION
**Pull request by erdOne**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/RingTheory/Kaehler.lean [hunk 1]
```diff
@@ -207,12 +207,12 @@ set_option linter.uppercaseLean3 false in
...
   { toLinearMap := KaehlerDifferential.DLinearMap R S
     map_one_eq_zero' := by
-      dsimp [KaehlerDifferential.DLinearMap_apply]
+      dsimp [KaehlerDifferential.DLinearMap_apply, Ideal.toCotangent_apply]
       congr
       rw [sub_self]
     leibniz' := fun a b => by
       have : LinearMap.CompatibleSMul { x // x ∈ ideal R S } (Ω[S⁄R]) S (S ⊗[R] S) := inferInstance
-      dsimp [KaehlerDifferential.DLinearMap_apply, - Ideal.toCotangent_apply]
+      dsimp [KaehlerDifferential.DLinearMap_apply]
```
Comment (grunweg) [inline Mathlib/RingTheory/Kaehler.lean:215]: I like this change!

File: Mathlib/RingTheory/Ideal/Cotangent.lean [hunk 2]
```diff
@@ -109,7 +109,7 @@ theorem to_quotient_square_comp_toCotangent :
   LinearMap.ext fun _ => rfl
 #align ideal.to_quotient_square_comp_to_cotangent Ideal.to_quotient_square_comp_toCotangent
 
--- @[simp] -- Porting note: not in simpNF
+@[simp]
```
Comment (grunweg) [inline Mathlib/RingTheory/Ideal/Cotangent.lean:112]: I presume you checked that this differs from the lemma that would be generated by simps.
(If not, I'd suggest replacing this by the auto-generated one.)

**Review by grunweg — APPROVED** [review]
Looks straightforward.

File: Mathlib/RingTheory/Ideal/Cotangent.lean (hunk 2 above)
Comment (erdOne) [reply Mathlib/RingTheory/Ideal/Cotangent.lean:112]: This is an ordinary simp lemma and simps cannot generate this.

Comment (grunweg) [reply Mathlib/RingTheory/Ideal/Cotangent.lean:112]: :bulb: Makes sense!

**Review by jcommelin — APPROVED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**